ecc_go/README.md

```diff
@@ -0,0 +1,84 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+# Go Chaincode Support for Fabric Private Chaincode
+
+This directory contains the components to enable go support Fabric Private Chaincode.
+
+## Overview
+ TODO
+ 
+In particular, it contains:
+- Go library to be included in a go chaincode
+- Building and packaging utilities
+
+### Architecture
+TODO
+
+## Install Ego inside dev environment
+
+This installation assumes a working FPC dev environment.
+You can find all setup information in the getting started section of our [README.md](../README.md).
+
+To start with we need to install the ego compiler in your dev environment. 
+Note that the FPC dev container comes with ego already installed.
+This is required to build and sign the FPC go chaincode.
+You can find more information about ego installation on the official [documentation](https://docs.edgeless.systems/ego/#/getting-started/install).
+
+Install ego by running the following:
+```bash
+wget -qO- https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add
+add-apt-repository "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu `lsb_release -cs` main"
+wget https://github.com/edgelesssys/ego/releases/download/v0.4.1/ego_0.4.1_amd64.deb
+apt install ./ego_0.4.1_amd64.deb build-essential libssl-dev
+```
+
+In order to package and run FPC go chaincode we use the `ccenv-go` docker image.
+Run the following to create the docker image.
+
+```bash
+cd $FPC_PATH/utils/docker/
+make ccenv-go
+```
+
+Now you have all you need to get started with your first FPC go chaincode. You can 
```

Comment:
@munapower see this

Reply:
First sentence says "This installation assumes a working FPC dev environment." So I went to the readme that is linked in the following line to get a working dev environment. So it should be made clear what steps of that readme should be completed because it is not all of them.